development/test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Build Jira config
+echo "${JIRA_TOKEN}" > /tmp/token
```

Comment:
`jira_config_gen` should be able to use `JIRA_TOKEN` from OS environment as well.

Reply:
I considered this but don't want to encourage that behavior as it would make it incredibly easy to leave the token in OCPCI.

Reply:
I prefer to give the option, all others can be set via OS environment
Up to you, you can resolve the conversion if you want to leave it as is.

Reply:
I think I'd like to err on the side of caution and avoide this. I don't want this tool to be the reason someone's Jira creds get leaked. We can revist this in the future though when the DPTP team has a better hold on filtering secrets in public logs.

cc: @ascerra as he has been working on something to help with this issue.